contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+
+    getPathsForAllAnswers() {
+        const allAnswerPaths = []
+        const allAnswers = []
+        const answersUsed = []
+
+        // Log ids of all answers in section
+        this.questions.forEach((question) => {
+            question.answers.forEach((answer) => {
+                allAnswers.push(answer.id)
+            })
+        })
+
+        // Log ids of answers used in minimum paths for recommendations
+        const allRecommendationPaths = this.minimumPathsForRecommendations;
+        for (const maturity in allRecommendationPaths) {
+            allRecommendationPaths[maturity].forEach((question) => {
```

Comment:
Similar thing here. Remove lines 173-179. Replace line 162 with 

```js
const answersUsed = Object.values(allRecommendationPaths).reduce((previous, current) => {
if(previous.indexOf(question.answer.id) == -1){
    previous.push(question.answer.id);
}

return previous;
}, []);
```

Reply:
I like this a lot, thanks.